src/main/java/com/hatchways/exception/InvalidDirectionException.java

```diff
@@ -0,0 +1,4 @@
+package com.hatchways.exception;
+
+public class InvalidDirectionException extends RuntimeException {
```

Comment:
Check out this thread on stackoverflow to confirm that you're correctly using `Exception` vs `RuntimeException`: https://stackoverflow.com/questions/19857008/extending-exception-runtimeexception-in-java